ROADMAP.md

```diff
@@ -31,38 +31,48 @@ The lifecycle events of a container also help identify areas to implement hooks
 
 *Owner:* mrunalp
 
-### Define Standard Container Actions
+### Define Standard Container Actions (Target release: v0.3.0)
 
 Define what type of actions a runtime can perform on a container without imposing hardships on authors of platforms that do not support advanced options.
 
-*Owner:*
+*Owner:* duglin
 
 ### Clarify rootfs requirement in base spec
 
 Is the rootfs needed or should it just be expected in the bundle without having a field in the spec?
 
+Resolution: keep it as is - user definable
+Submit as a different PR
+For reference: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/6ZKMNWujDhU
+
 *Owner:*
 
 ### Container Definition
 
 Define what a software container is and its attributes in a cross platform way.
 
-*Owner:*
+Could be solved by lifecycle/ops and create/start split discussions
+
+*Owner:* vishh & duglin
 
 ### Live Container Updates
 
 Should we allow dynamic container updates to runtime options?
 
-*Owner:* vishh
+Proposal: make it an optional feature
+
+*Owner:* hqhq (was vishh) robdolinms, bcorrie
 
 ### Protobuf Config
 
 We currently have only one language binding for the spec and that is Go.
 If we change the specs format in the respository to be something like protobuf then the generation for multiple language bindings become effortless.
 
+Not going to do it
```

Comment:
On Wed, Jan 13, 2016 at 12:11:23PM -0800, Vincent Batts wrote:
> That's what we're working on right now. :-)

Excellent :).  I don't see the need to land #300 with WIP wording and
follow-up PRs, but as long as the motivation gets documented
somewhere, I'm happy.